From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
Date: Tue, 12 Apr 2022 12:24:13 +0530
Subject: [PATCH] update the value_model for ADL

update value_model=`grep -i model /proc/cpuinfo
| cut -d ':' -f2 | head -n 1`

Tracked-On: HSD-16016177995
Signed-off-by: Kothapeta, BikshapathiX <bikshapathix.kothapeta@intel.com>
---
 groups/codecs/configurable/auto_hal.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/groups/codecs/configurable/auto_hal.in b/groups/codecs/configurable/auto_hal.in
index eab9dfb..281a348 100644
--- a/groups/codecs/configurable/auto_hal.in
+++ b/groups/codecs/configurable/auto_hal.in
@@ -9,6 +9,12 @@ if [ $value_model == 166 ]; then
 elif [ $value_model == 141 ]; then
     setprop ro.vendor.media.target_variant "_gen12"
     setprop ro.vendor.media.target_variant_platform "_tgl"
+elif [ $value_model == 151 ]; then
+    setprop ro.vendor.media.target_variant "_gen12"
+    setprop ro.vendor.media.target_variant_platform "_adl"
+elif [ $value_model == 154 ]; then
+    setprop ro.vendor.media.target_variant "_gen12"
+    setprop ro.vendor.media.target_variant_platform "_adl"
 else
     :
 fi
-- 
2.17.1

